lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -14,6 +14,19 @@ defmodule ArchEthicWeb.FaucetController do
 
   @pool_seed Application.compile_env(:archethic, __MODULE__)[:seed]
```

Comment:
I think it may raise an error, if you change on runtime the `enabled` flag from env variable, because the `@pool_seed` is from compilation env.

Reply:
If I use get_env instead, credo throws error.

Reply:
you can use the specific key to load from compilation: `@pool_seed Application.compile_env(:archethic, [__MODULE__, :seed])`